src/microbe_stage/Microbe.cs

```diff
@@ -753,8 +774,22 @@ public void PlaySoundEffect(string effect)
         // TODO: make these sound objects only be loaded once
         var sound = GD.Load<AudioStream>(effect);
 
-        otherAudio.Stream = sound;
-        otherAudio.Play();
+        // Find a player not in use or create a new one if none are available.
+        var player = otherAudioPlayers.Find(nextPlayer => !nextPlayer.Playing);
+
+        if (player == null)
```

Comment:
I really think there should be a hard limit like 10 to prevent bugs where a cell is playing 10000 sounds at once each frame.

Reply:
Problem with that is what do you do if you end up with that many? Just not play the sound if they're all busy? I could add a limit if you're that concerned about it.

Reply:
I would just not play the sound. And I'd like to have a limit.
Many audio engines have an internal max limit for active sounds, and they just stop the quietest sound or not play new sounds if that limit is hit.
If you have 10 sound effects coming from a cell at once it will be chaotic enough already that the player won't notice if some sound isn't played.